### PR TITLE
CotE: Fire Tensai Acolyte

### LINKED ISSUE #2749: CotE - Fire Tensai Acolyte
image

### PR DESCRIPTION
**Pull request by Kwaice**
Closes #2749

### CODE REVIEW

File: server/game/cards/06-CotE/FireTensaiAcolyte.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const DrawCard = require('../../drawcard.js');
+
+class FireTensaiAcolyte extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: context => context.player.opponent && !this.game.isDuringConflict('fire'),
```
Comment (jeremylarner) [inline server/game/cards/06-CotE/FireTensaiAcolyte.js]: why do you have `context.player.opponent` here?

File: test/server/cards/06-CotE/FireTensaiAcolyte.spec.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                });
+                expect(this.game.currentConflict.attackers).toContain(this.masahiro);
+                expect(this.game.currentConflict.attackers).toContain(this.acolyte);
+            });
+
+            it('should be able to participate in fire conflicts if moved in', function() {
+                this.noMoreActions();
+                this.initiateConflict({
+                    type: 'military',
+                    ring: 'fire',
```
Comment (jeremylarner) [inline test/server/cards/06-CotE/FireTensaiAcolyte.spec.js]: Wouldn't it make more sense to test whether it can participate in non-fire conflicts?

Comment (Kwaice) [reply test/server/cards/06-CotE/FireTensaiAcolyte.spec.js]: Ah yes. That's a mistake haha.

File: server/game/cards/06-CotE/FireTensaiAcolyte.js (hunk 1 above)
Comment (Kwaice) [reply server/game/cards/06-CotE/FireTensaiAcolyte.js]: I thought that was required for single player games or something like that.

**Pull request merged**